artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/Infrastructure/ScrumServiceHttpClient/ProjectHttpClientWrapper.cs

```diff
@@ -30,15 +30,30 @@ internal class ProjectHttpClientWrapper : IProjectHttpClientWrapper
             refreshToken,
             (httpClient, user) => httpClient.GetAsync(urlFactory(user)));
 
-    public async Task<TResponse> PostHttpRequest<TRequest, TResponse>(
+    public async Task<TResponse> ResourceUpdateHttpRequest<TRequest, TResponse>(
+        ResourceUpdateHttpMethod httpMethod,
```

Comment:
a po co tak? Mogłeś dorzucić po prostu kolejną metodę która to rozszerzy zamiast modyfikować istniejącą i za dużo by się nie różniło a - OCP